mobile/src/modal.js

```diff
@@ -46,6 +46,18 @@ export class CustomModal extends Component {
             height: this.state.height,
             marginTop: this.state.marginTop
         }
+        
+        const androidStyle = {
```

Comment:
`androidStyle` can just contain only the additional padding styles, since line 95 already includes both `newStyle` and `androidStyle`

Reply:
Great point. Changes added.